### DISCUSSION TITLE
Guido Rosso - Rive

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#341] [Announcements]
- https://www.linkedin.com/in/guidorosso/
- https://x.com/guidorosso
- https://dribbble.com/guidorosso

## Questions

- [x] What is Rive?
- [x] Were you inspired by flash?
- [x] What do you think the key to making a delightful animation is?
- [x] What does the workflow look like for creating these animations?
- [x] How do you keep animations performance across multiple platforms?
- [x] Is it something that replaces tools like figma? Is it a full fledged canvas editor or are you expected to make the assets beforehand?
- [x] What makes it better than other animation techniques like Lottie?
- [x] What is your favorite primitive the platform offers?
- [x] What do state machines enable you to build?
- [x] What runtimes does the engine support? Do you know of the tech that makes it possible?
- [x] How are people using Rive for game dev? What makes it different than other game engines?
- [x] What have some of the harder problems been while building this product?

### Future

- [x] What do you think the future of UI development looks like?

**Comment by just-be-dev** [comment]
Cover:
- Editor: Statemachine driven animations
- Runtimes: Cross platform, open source, small file size
- Renderer

> we can’t quickly implement highly requested features (like blurs and drop shadows) since not all renderers support them.
